_episodes/02-filedir.md

```diff
@@ -607,18 +607,18 @@ Run `pwd` and `ls -F` to ensure that we're in the directory we expect.
 > The shell interprets the character `~` (tilde) at the start of a path to
 > mean "the current user's home directory". For example, if Nelle's home
 > directory is `/Users/nelle`, then `~/data` is equivalent to
-> `/Users/nelle/data`. This only works if it is the first character in the
+> `/Users/nelle/data`. The tilde only works if it is the first character in the
```

Comment:
It's the expansion of the tilde which doesn't work in this scenario. I don't think this edit is an upgrade.